bin/test-with-real-projects.sh

```diff
@@ -30,13 +30,20 @@ proxymanager)
 	~/project/psalm --monochrome
 	;;
 
+psl)
+	git clone git@github.com:azjezz/psl.git
+	cd psl
+	composer install --ignore-platform-reqs
```

Comment:
ext-bcmath is required, but it is missing from CI.

Reply:
@muglug build seems to pass as psalm uses the stubs for bcmath, do you think this is fine or should i add ext-bcmath to the CI?

Reply:
Tests passed, so I'm happy